src/repositories/addStoreMission.repository.js

```diff
@@ -0,0 +1,15 @@
+import { prisma } from "../db.config.js";
+
+export const addStoreMission = async (data) => {
+  const created = await prisma.mission.create({
+    data: {
+      regionId: data.region_id ? BigInt(data.region_id) : null,
+      storeId: BigInt(data.store_id),
+      title: data.title || null,
```

Comment:
title에 null을 넣게 허용하는건 요구사항에 맞지 않는 것 같습니다!
```suggestion
      title: data.title || null,
```